lineman/app/components/group_page/discussions_card/discussions_card.scss

```diff
@@ -1,3 +1,11 @@
+.discussions-card{
+  @include cardNoPadding;
+}
+
+.discussions-card__header{
```

Comment:
.lmo-card-heading-padding doesn't quite work because it puts too much padding on the bottom.